src/blocked.c

```diff
@@ -166,6 +167,7 @@ void unblockClient(client *c) {
     } else if (c->btype == BLOCKED_PAUSE) {
         /* Mark this client to execute its command */
         c->flags |= CLIENT_PENDING_COMMAND;
+        listDelNode(server.paused_clients,c->paused_list_node);
```

Comment:
maybe you better nullify paused_list_node after doing that.

Reply:
Good point.